test/resources/firebase_config_bad_key.json

```diff
@@ -0,0 +1,4 @@
+{
+  "databaseUrl": "https://hipster-chat.firebaseio.mock",
```

Comment:
Please add another key that is obviously bad. Something like `"notSupported": "some value"`